style.css

```diff
@@ -1081,6 +1233,45 @@ form button:focus {
 }
 
 @media all and (max-width: 768px) {
+  /* modal and pop part start */
+  .container-modal .modal {
+    justify-content: space-between;
+    top: 60px;
+  }
+
+  .modal-desc {
+    flex-direction: column;
+    width: 100%;
+    height: 70%;
+    justify-content: space-between;
+    gap: 1px;
+    margin-top: 2px;
+  }
```

Comment:
- [ ] Good job so far, but you still have issues to fix with your design. Your design should stick as much as possible to the figma design. As mentioned by the previous reviewer your popup should be responsive also your design should much to the figma design 🙃 🙃 

Your design

<img width="349" alt="Screen Shot 2023-02-09 at 00 04 00" src="https://user-images.githubusercontent.com/74501523/217652188-a159c049-15fe-45ef-b8c1-7457a290d82d.png">

Figma design

<img width="289" alt="Screen Shot 2023-02-09 at 00 05 03" src="https://user-images.githubusercontent.com/74501523/217652244-4c1c9887-2af6-45a9-ae86-c87ea4490626.png">